app/editor/src/features/admin/products/ProductFilter.tsx

```diff
@@ -26,17 +32,31 @@ export const ProductFilter: React.FC<IAdminFilterProps> = ({ onFilterChange, onS
           <IconButton
             iconType="search"
             onClick={() => {
-              onSearch?.(filter);
+              onSearch?.(filter, isSubscribedId);
             }}
           />
         )}
       </Text>
+      {productId && (
+        <Checkbox
+          name="isSubscribed"
+          label="Is subscribed"
+          className="checkbox-filter"
+          checked={!!isSubscribedId}
+          onChange={(e) => {
+            const s = e.target.checked ? productId : undefined;
+            setIsSubscribedId(s);
+            onSearch?.(filter, s);
+          }}
+        />
+      )}
       <IconButton
         iconType="reset"
         onClick={() => {
+          setIsSubscribedId(undefined);
           setFilter('');
           onFilterChange?.('');
-          onSearch?.('');
+          onSearch?.('', undefined);
```

Comment:
You shouldn't need to do this if the argument is optional.  But it isn't an issue to do this.